doc/03-omnifest/01-directive.md

```diff
@@ -196,26 +196,66 @@ otk.meta.kiwi:
 
 ## `otk.customization`
 
+A `otk.customization` is distinct from a `otk.define` because:
+- End-users will supply them
+- ?
+
 Customizations are conditional blocks that receive separate input through
-`otk compile -Cname=data`, a customization is considered to be active when it
-is passed data. If a customization is passed multiple times then the `defined`
-block is repeated multiple times, once for each input.
+a `customization` file `otk compile -C customizations.yaml` that contains a
+map of customization settings in yaml format, e.g.
```

Comment:
You would have to specify this format further. Customizations should be able to be passed multiple times (e.g: files).

Reply:
Thanks, we could allow multple -C that are merged (just like the previous specification). Whats the concern about:
```yaml
file:
 - path: /path/1
   user: ...
 - path: /path/2
 ``` 
i.e. having them in the same customization.yaml ? Lack of flexibility?